docs/content/ecs/logging.md

```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, `LogType.Log` is not sent to SpatialOS console. This can be changed by instantiating the `ForwardingDispatcher` with a different `minimumLogLevel` parameter.
```

Comment:
the first sentence should be rephrased a bit. maybe something like:

Note: by default, messages with log level `LogType.Log` are not send to SpatialOS.